composer.json

```
@@ -76,15 +77,15 @@
     "ramsey/uuid": "^3.8",
     "relay/relay": "~2.0",
     "composer-runtime-api": "^2.0",
-    "ext-pdo": "*"
+    "ext-pdo": "*",
```

```suggestion
    "ext-pdo": "*"
```
looks like not valid json